docs/API/V1/contents.md

```diff
@@ -33,25 +33,23 @@ Either the `<dataset>` or the relative `<path>` within the dataset does not exis
 
 ## Response body
 
-This returns an `application/json` response body describing the target of `<path>`, in terms of "file" and "directory" JSON objects.
+This API returns an `application/json` response body consisting of a JSON object which describes the target `<path>`, in terms of lists of "file" and "directory" JSON objects (as described below).
 
-When the target is a directory, the response object contains a list of subdirectories and a list of files; if the target is a file (either a regular file or a symlink) it returns the file object for that path.
-
-When the target directory contains symlinks, the symlinks will be reported either in the `"file"` list (if the target of the symlink is a file) or in the `"directory"` list (if the target of the symlink is a directory).
+When the target is a directory, the response object contains a list of subdirectories and a list of files; if the target is a file it returns the file object for that path.
 
 ### Directory
 
-When the `<path>` refers to a directory within the dataset representation, Pbench returns a JSON object with a list of subdirectory objects and a list of file objects.
+When the `<path>` refers to a directory within the dataset representation, Pbench returns a JSON object containing a list of subdirectory objects (`directories`) and a list of file objects (`files`).
```

Comment:
`directories` and `files` are keys, right?  So, should the be in quotes?  E.g., `"directories"`?

Reply:
Since the text says that Pbench returns a JSON object, I would refer to [this definition](https://www.json.org/json-en.html), which says that object members are a "string" followed by a colon followed by a "value" (with optional whitespace sprinkled in there), and a "string" is "characters" enclosed in double-quotes.

But, yeah, if someone is implementing code around this, they are probably using Javascript or Python, in which case there is additional or different syntax involved...but, then we would be talking about Javascript or Python and not about JSON.  😉